src/devcenter/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '0.1.2'
+VERSION = '0.2.0'
```

Comment:
For GA release, please use version '1.0.0'

Reply:
Fixed